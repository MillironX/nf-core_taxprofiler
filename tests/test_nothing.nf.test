nextflow_pipeline {

    name "Test pipeline: NFCORE_TAXPROFILER"
    script "main.nf"
    tag "pipeline"
    tag "nfcore_taxprofiler"
    tag "nfcore_taxprofiler_test_nothing"
    tag "test_nothing"

    test("nf-core/taxprofiler test_nothing profile") {
        when {
            params {
                outdir = "$outputDir"
            }
        }

        then {
            assertAll(
                { assert workflow.success },
                { assert snapshot(
                        file("${params.outdir}/fastqc").listFiles().exists,
                        file("${params.outdir}/multiqc").listFiles().exists,
                    ).match()
                }
            )
        }
    }
}
