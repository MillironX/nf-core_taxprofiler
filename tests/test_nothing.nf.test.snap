{
    "nf-core/taxprofiler test_nothing profile": {
        "content": [
            [
                [
                    [
                        "2612_ERR5766176_B_raw_1_fastqc.html:md5,22c25a2cb921172db84a7b86cc125b04",
                        "2612_ERR5766176_B_raw_1_fastqc.zip:md5,b8b47d68603cb13cab569f4497dfc11e",
                        "2612_ERR5766176_B_raw_2_fastqc.html:md5,49c94b6be3881278c943dcf8b2fb348f",
                        "2612_ERR5766176_B_raw_2_fastqc.zip:md5,b3de30c18440c9f079037ee75e1eefbe",
                        "2612_ERR5766176_raw_1_fastqc.html:md5,aeadc849f7871335c731172951de743f",
                        "2612_ERR5766176_raw_1_fastqc.zip:md5,7103d51310f0eb1af777aa8794d4dbcd",
                        "2612_ERR5766176_raw_2_fastqc.html:md5,ae9510a231479a459162ba2c43487a56",
                        "2612_ERR5766176_raw_2_fastqc.zip:md5,a53c752eb7b9fd963b195b30ff4d8147",
                        "2612_ERR5766180_raw_fastqc.html:md5,579396567aa78671d5038f6bb6f7cbba",
                        "2612_ERR5766180_raw_fastqc.zip:md5,40051d62da8e8e167b4495781914ea85",
                        "2613_ERR5766181_raw_1_fastqc.html:md5,7164c1f3d4db16c85bc98f6960f6f731",
                        "2613_ERR5766181_raw_1_fastqc.zip:md5,a9253a70e136ac0bf1ee352d9f777eaa",
                        "2613_ERR5766181_raw_2_fastqc.html:md5,c413fa16d6226aeeb8e11b49a0a036b9",
                        "2613_ERR5766181_raw_2_fastqc.zip:md5,eea822bcf1f73387a7ec49a8890772bc",
                        "ERR3201952_ERR3201952_raw_fastqc.html:md5,b1756e66fd863ee606a8c91d439f8775",
                        "ERR3201952_ERR3201952_raw_fastqc.zip:md5,7a8e4d6101d38b45b360308786c300fb"
                    ]
                ],
                [
                    [
                        "mqc_fastqc_adapter_content_plot_1.txt:md5,03fcfd4ace824968a095e82a042ec82d",
                        "mqc_fastqc_overrepresented_sequences_plot_1.txt:md5,c8f294c7217e2e54117bce92b423ad2a",
                        "mqc_fastqc_per_base_n_content_plot_1.txt:md5,1f78a15290cce751b9d1c76310fb9987",
                        "mqc_fastqc_per_base_sequence_quality_plot_1.txt:md5,c1876d03c7d0643f6a8d9bc14853cbc4",
                        "mqc_fastqc_per_sequence_gc_content_plot_Counts.txt:md5,1b8ee374b085e3e2a423e90d7c49d61c",
                        "mqc_fastqc_per_sequence_gc_content_plot_Percentages.txt:md5,561610c73e18af864207d5938395c095",
                        "mqc_fastqc_per_sequence_quality_scores_plot_1.txt:md5,e60e51c43c71ddc2e1221680723839e8",
                        "mqc_fastqc_sequence_counts_plot_1.txt:md5,004796a97685bdeb02292be79e090673",
                        "mqc_fastqc_sequence_duplication_levels_plot_1.txt:md5,71002c0f9191ea67ed9d245b308c0f04",
                        "mqc_fastqc_sequence_length_distribution_plot_1.txt:md5,2e33e514ed41a7566fd8af34cfe49334",
                        "multiqc.log:md5,543a8c1ccfe2218d11d138fab94ee70f",
                        "multiqc_citations.txt:md5,4c806e63a283ec1b7e78cdae3a923d4f",
                        "multiqc_data.json:md5,4b3a3f505f4b7a435788fa24cabea03d",
                        "multiqc_fastqc.txt:md5,4e145ddc71fb1b37b62df907fa452200",
                        "multiqc_general_stats.txt:md5,2ff3830e0098393fc60d2e9292e2ac66",
                        "multiqc_software_versions.txt:md5,a8c72f50c3dbc98152e15bcba21dbf7f",
                        "multiqc_sources.txt:md5,170f76bce159e4f5a47e586fcf934f22"
                    ],
                    [
                        [
                            "mqc_fastqc_adapter_content_plot_1.pdf:md5,92061915b9c94eafb383fb62bcf53d67",
                            "mqc_fastqc_overrepresented_sequences_plot_1.pdf:md5,f099c363d18f8495ab24e6106a13f95d",
                            "mqc_fastqc_per_base_n_content_plot_1.pdf:md5,7a2ed304e10fe60e0d2c003cb3695d22",
                            "mqc_fastqc_per_base_sequence_quality_plot_1.pdf:md5,cf98a0065e78f9de9e9820c48c72ea28",
                            "mqc_fastqc_per_sequence_gc_content_plot_Counts.pdf:md5,4fd1162de6ed98d9c8979c363c8c6d8d",
                            "mqc_fastqc_per_sequence_gc_content_plot_Percentages.pdf:md5,ef3fd50c5ad666f5c9d8b7272bb63ffe",
                            "mqc_fastqc_per_sequence_quality_scores_plot_1.pdf:md5,1ed5ff1af74249601fbbedaf0896eeec",
                            "mqc_fastqc_sequence_counts_plot_1.pdf:md5,05699a06111266ba1a2fc6c5b6225b5e",
                            "mqc_fastqc_sequence_counts_plot_1_pc.pdf:md5,1e12f97bbc1d11778ff021e273ed9306",
                            "mqc_fastqc_sequence_duplication_levels_plot_1.pdf:md5,da78092e24049610606c5a9f08022730",
                            "mqc_fastqc_sequence_length_distribution_plot_1.pdf:md5,b5fe60395c94ea578a608abdcc2c879a"
                        ],
                        [
                            "mqc_fastqc_adapter_content_plot_1.png:md5,d0944b5f795c6538ff950131907a235c",
                            "mqc_fastqc_overrepresented_sequences_plot_1.png:md5,215518a939fbdd13b440100c2dfee886",
                            "mqc_fastqc_per_base_n_content_plot_1.png:md5,d8836c33718bededd2d27031412cef1a",
                            "mqc_fastqc_per_base_sequence_quality_plot_1.png:md5,18ce1e486ebeb0050abb027f591dec1e",
                            "mqc_fastqc_per_sequence_gc_content_plot_Counts.png:md5,3e26a0f31a552797446c1a2da93c8906",
                            "mqc_fastqc_per_sequence_gc_content_plot_Percentages.png:md5,9540195d9eaf7c503b6dc9f6dd1b65ae",
                            "mqc_fastqc_per_sequence_quality_scores_plot_1.png:md5,5a132a8645b54d6c7665e5ce27b382c5",
                            "mqc_fastqc_sequence_counts_plot_1.png:md5,4a5d277e9dfdab7539149f463e3cd3ff",
                            "mqc_fastqc_sequence_counts_plot_1_pc.png:md5,cc0518d03f7a5bfbee3ff1d5c70e08c5",
                            "mqc_fastqc_sequence_duplication_levels_plot_1.png:md5,d95749003f744f1149c79a1b477a5323",
                            "mqc_fastqc_sequence_length_distribution_plot_1.png:md5,24bf7cfd1113bc12a3eb311ec2642a51"
                        ],
                        [
                            "mqc_fastqc_adapter_content_plot_1.svg:md5,975bf3c4b47ed61a6fea1a5613cc4856",
                            "mqc_fastqc_overrepresented_sequences_plot_1.svg:md5,318a787111dfd53f1ab052a19e0c61b7",
                            "mqc_fastqc_per_base_n_content_plot_1.svg:md5,aabb76bd8d048e5189be0647138fa16e",
                            "mqc_fastqc_per_base_sequence_quality_plot_1.svg:md5,101f0c24b9993719f0dd0aa33d4d3358",
                            "mqc_fastqc_per_sequence_gc_content_plot_Counts.svg:md5,7519277c6d46ccdc33613655c55cdecc",
                            "mqc_fastqc_per_sequence_gc_content_plot_Percentages.svg:md5,bebd34f4f601a49473ffac0ad86f0549",
                            "mqc_fastqc_per_sequence_quality_scores_plot_1.svg:md5,6ab7712be2dc346c4125ca24bc676023",
                            "mqc_fastqc_sequence_counts_plot_1.svg:md5,de6647727b404111cc585bc130fd609b",
                            "mqc_fastqc_sequence_counts_plot_1_pc.svg:md5,3748092488ed6c2e5cda3c0c47f517d3",
                            "mqc_fastqc_sequence_duplication_levels_plot_1.svg:md5,691ac9510e93e27ed61f6cf56d4187ed",
                            "mqc_fastqc_sequence_length_distribution_plot_1.svg:md5,9b80a697ccb105a09e4b2afc2953cd44"
                        ]
                    ],
                    "multiqc_report.html:md5,b3a306f35034be4de49257fee9067e8f"
                ],
                [
                    "execution_report_2023-12-01_05-58-24.html:md5,0ddb6c350255ae5537e6d304d573ae45",
                    "execution_timeline_2023-12-01_05-58-24.html:md5,14a213458c0d36187c3b18d942e42732",
                    "params_2023-12-01_05-59-25.json:md5,acae7d15c24ee4cc322b6bf99fcce9eb",
                    "pipeline_dag_2023-12-01_05-58-24.html:md5,e3b835328ae04fe552aaf94dffdc430c",
                    "software_versions.yml:md5,a24a6dddf273082d065dea785bfe5f11"
                ]
            ]
        ],
        "timestamp": "2023-12-01T05:59:25.966329598"
    }
}